web/webkit/src/main/scala/net/liftweb/http/provider/HTTPProvider.scala

```diff
@@ -87,12 +87,29 @@ trait HTTPProvider {
         preBoot
         b.boot
       } catch {
+        // The UnavailableException is the idiomatic way to tell a Java application container that
+        // the boot process has gone horribly, horribly wrong. That _must_ bubble to the application
+        // container that is invoking the app. See https://github.com/lift/framework/issues/1843
+        case unavailableException: javax.servlet.UnavailableException =>
+          logger.error("------------------------------------------------------------------")
+          logger.error("------------------------------------------------------------------")
+          logger.error("------------------------------------------------------------------")
+          logger.error("------------------------------------------------------------------")
+          logger.error("********** Failed to Boot! An unavailable exception was thrown an all futher boot activities are aborted", unavailableException);
```

Comment:
I might say “An UnavailableException was thrown”; either way, it should be “and” rather than “an”.